### PR TITLE
add solved count and ranks for current and last week

### PR DESCRIPTION
**Pull request by lukashass**
Part of https://github.com/streetcomplete/StreetComplete/issues/2946

Adds following statistics:

- solved quest count for current week
- solved quest count for last week
- global rank for current week
- global rank for last week
- country rank for current week
- country rank for last week

This is currently quite copy-paste heavy which I don't really like but templating in the SQL wouldn't be particularly pretty either.

## Off-topic

Moves procedure creation to a separate script for easier setup.

### CODE REVIEW

**Comment by matkoniecz** [comment]
@lukashass Are you planning to complete/update it?

**Comment by lukashass** [comment]
@matkoniecz I don't have that much time ATM. Feel free to continue this :+1:

File: README.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ Tested with PHP 7.3 and MySQL 5.6.
...
 - Create the respective database
+- Run `create_rank_procedures.php` before the first exection of `generate_ranks.php`
 - `delete_data_of_deleted_users.php` should be scheduled to run daily, `update_incomplete_statistics.php` should be scheduled to run in regular intervals, for example every minute or so, `generate_ranks.php` should be scheduled to run in regular intervals, hourly is good, but can be in shorter intervals as well
 
 ## Development
 
 - Open project in [devcontainer](https://code.visualstudio.com/docs/remote/containers-tutorial)
 - Create `config.php` from `config.sample.php` template and fill with development settings
 - For debugging set a breakpoint and run the `Listen for Xdebug` launch config before running any scripts
+- Run `create_rank_procedures.php` before the first exection of `generate_ranks.php`
```
[the remark this answers is not in the document]
Comment (lukashass) [reply README.md:19]: Sure, I just wasn't sure about the overhead. Maybe it is even possible to check whether the procedure already exists?

[the remark this answers is not in the document]
Comment (westnordost) [reply README.md:19]: Probably, but the source code for the procedure may have changed from deployment to deployment.

**Comment by westnordost** [comment]
Superceded by https://github.com/streetcomplete/sc-statistics-service/pull/24 - too many merge conflicts (and open issues) on this one to base the work on it.

**Pull request closed without merging**